lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs

```diff
@@ -527,21 +527,13 @@ encryptPassphrase  (Passphrase bytes) = do
         <> BA.convert @ByteString (fastPBKDF2_SHA512 params bytes salt)
 
 -- | Manipulation done on legacy passphrases before getting encrypted.
---
--- Yes there's CBOR. Yes it's hashed before being encrypted. You don't actually
--- pronounce the 'l' in salmon. A lot of things in the world don't make sense
```

Comment:
I will miss this comment ;-)

Reply:
Me too. It'll live in the git history and will be our little secret 🤫